### ISSUE TITLE
与 Nova 插件的兼容问题

### ISSUE THREAD

**Issue by postyizhan** [Dreeam-qwq/Leaf#75]
# 版本
均为 1.20.4
# paper
![图片](https://github.com/Winds-Studio/Leaf/assets/97342038/b47444bc-1259-42e6-9186-0993d0df52bb)
正常加载
# leaf
![图片](https://github.com/Winds-Studio/Leaf/assets/97342038/7558a273-36ad-45d0-8166-74cce4985d1c)
![图片](https://github.com/Winds-Studio/Leaf/assets/97342038/a93fcd7e-c5d7-446e-b178-08ac9af0c55d)
![图片](https://github.com/Winds-Studio/Leaf/assets/97342038/4b681b2d-8a32-43cf-8a4a-21d8001a4628)
![图片](https://github.com/Winds-Studio/Leaf/assets/97342038/d62c3d72-5b2d-4d9e-b0ff-7c9a8bd54e16)
![图片](https://github.com/Winds-Studio/Leaf/assets/97342038/4d3f0fc1-07d2-4c3d-bb8e-344eda371f1b)

**Comment by TheFloodDragon** [comment]
Nova兼容性就这样

只能让Nova将Leaf识别成Purpur

修改代码:
[nova/src/main/kotlin/xyz/xenondevs/nova/util/ServerUtils.kt](https://github.com/xenondevs/Nova/tree/main/nova/src/main/kotlin/xyz/xenondevs/nova/util/ServerUtils.kt)

```Kotlin
val SERVER_SOFTWARE by lazy {
    when (Bukkit.getVersion().substringAfter('-').substringBefore('-')) {
        "Bukkit" -> CRAFT_BUKKIT
        "Spigot" -> SPIGOT
        "Paper" -> PAPER
        "Pufferfish" -> PUFFERFISH
        "Tuinity" -> TUINITY
        "Purpur" -> PURPUR
        "Airplane" -> AIRPLANE
         "Leaf" -> PURPUR // !!!此处为添加内容!!! 只能让Leaf先被判为 Purpur
         else -> UNKNOWN
    }
}
```

**Comment by Dreeam-qwq** [comment]
Fixed in b80c8b15d64a774fb7e9ffe6518074f92b4e7acc

In leaf config, set `fix_nova_compatibility.enabled` to true to enable this.
